### ISSUE TITLE
Upgrade to polkadot 2409

### ISSUE THREAD

**Issue by jak-pan** [galacticcouncil/hydration-node#946]
Deps 
polkadot - https://github.com/paritytech/polkadot-sdk/releases/tag/polkadot-stable2409-1
frontier - https://github.com/moonbeam-foundation/frontier/tree/moonbeam-polkadot-stable2409
orml is still on 2407 but should be simple upgrade - https://github.com/open-web3-stack/open-runtime-module-library/releases

**Comment by Roznovjak** [comment]
@jak-pan you are right, latest orml release uses 2407, but there is already opened [PR](https://github.com/open-web3-stack/open-runtime-module-library/pull/1011) for 2409 . I think we can use it.

**Comment by Roznovjak** [comment]
The upgrade has been merged in the `master`, so closing this one.
